nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractJsonQueryElasticsearch.java

```diff
@@ -105,6 +105,10 @@
     private String splitUpAggregations;
     private boolean outputNoHits;
 
+    public boolean getOutputNoHits() {
```

Comment:
package-privte would be fine, not `public`

Personally I'd move the method lower down the class too rather than putting it in the middle of the class field definitions... but that's more a personal preference than anything so happy to ignore that